lib/kitchen/directions/bake_index/v2.rb

```diff
@@ -62,8 +49,7 @@ class IndexSection
       def initialize(name:)
         @force_first = name == I18n.t(:eob_index_symbols_group)
         @name = name
-        collator = TwitterCldr::Collation::Collator.new(:pl)
-        @items = SortedSet.new { |a, b| collator.compare(a, b) }
+        @items = SortedSet.new
```

Comment:
I'm not sure that `SortedSet` takes a block in its constructor, that may have been the root of the ArgumentError.

Reply:
That was mine prediction, that I'm trying to compare elements in a wrong place :thinking:  I couldn't figure out where I should do this step.

Thank you for helping me with this! :relaxed: